### PR TITLE
Refactor build scripts

### PR DESCRIPTION
**Pull request by asanzo**
This change is motivated by the need to put Pilas Bloques in a docker container.

However, _you won't be seeing any Docker code here_, It's in another private repo. What we are doing here is **releasing a tar.gz file**. What the [other project](https://github.com/fundacion-sadosky/containerization/tree/prod/pilasbloques) does is download latest tar.gz, unzips, packs an NGinX Docker image, puts the contents there, ready to be run.

So what we are doing here is:

- Now every time we tag this repo, Travis builds the `tar.gz` and publishes in [releases](https://github.com/Program-AR/pilas-engine-bloques-website/releases). The `tar.gz` has _everything needed to deploy the site_: that is, Pilas Bloques site, and inside "online" folder, Pilas Bloques application. Anyone wanting to run Pilas Bloques frontend just needs to unzip and serve.
- Pilas Bloques is now not needed to be in a parallel folder anymore. Now Travis clones Pilas Bloques project inside this folder, in order to be able to put the `tar.gz` together.
- Now most of the building is performed in npm scripts rather than Makefile. Please read the README.md changes to know what are the available scripts.
- In the **Makefile** now only remain those scripts which are needed to publish Pilas Bloques in the current -soon to be deprecated- production server. When we switch servers, Makefile can go bye bye kiss we won't miss you.

### CODE REVIEW

**Review by la-silvestri — APPROVED** [review]
Gran trabajo @asanzo!, a veces pienso que me gustaría sacar algunos scripts del makefile y pasarlo a scripts sh y no seguir usando para todo el makefile... pero esta bien.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 [![Build Status](https://travis-ci.org/Program-AR/pilas-engine-bloques-website.svg?branch=master)](https://travis-ci.org/Program-AR/pilas-engine-bloques-website)
+
+## Commands
+
+- `npm install` will install dependencies and clone subproject pilas-bloques.
+- `npm test` runs tests.
+- `npm start` will serve the webpage.
+- `npm run build` will build the site to `dist` folder.
+- `npm run pb:updatedownloadlinks` will get subproject pilas-bloques' package.json version and date and update this site's files.
+- `npm run package:site` will put the site AND Pilas Bloques app in a nice ready-to-publish compressed tar.gz file. Gets Pilas Bloques from `master` branch.
```
Comment (PalumboN) [inline README.md:14]: Está bien que siempre trabaje con master?? No debería hacerlo con el último release?

(Igual entiendo que mucho no afecta, porque nosotros desarrollamos sobre develop).

**Review by PalumboN — APPROVED** [review]
image

File: README.md (hunk 1 above)
Comment (asanzo) [reply README.md:14]: https://github.com/Program-AR/pilas-bloques/issues/695

Comment (asanzo) [reply README.md:14]: ( o sea, no lo pensé, pero creo que es lo mismo hacerlo o no hacerlo)

**Pull request merged**